app/src/main/java/net/gsantner/markor/model/Document.java

```diff
@@ -227,7 +238,11 @@ public synchronized String loadContent(final Context context) {
                 content = "";
             }
         } else {
-            final Pair<String, GsFileUtils.FileInfo> result = GsFileUtils.readTextFileFast(_file);
+            // We try to load 2x. If both times fail, we return null
+            Pair<String, GsFileUtils.FileInfo> result = GsFileUtils.readTextFileFast(_file);
+            if (result.second.ioError) {
+                result = GsFileUtils.readTextFileFast(_file);
```

Comment:
I get this bug once every couple weeks where only the file will only partially load. This is an attempt to mitigate that.